src/util.jl

```diff
@@ -58,6 +58,39 @@ function update_tableau(state::StabilizerState)
     state.is_updated = true
 end
 
+function Base.rand(::Type{StabilizerState}, qubits::Int)
```

Comment:
I wasn't sure how to test this - are there any validity checks for StabilizerStates that could be run?
I may need to ask Madhav about what constraints there are for valid StabilizerStates.

I'll add a doc line - it's just a method of the Base rand function, so it will have documentation via ? already, but that can be extended.

Reply:
ad tests – not to block ourselves too much with that, for now some rudimentary tests like "test if it actually has the number of qubits I requested", or "check if when it's big enough it actually contains both X, Zs and all the phases", something along these lines?

ad docs – oh, indeed, didn't notice it extends the base function. In such case I guess a one-liner that explains it shortly is enough.